extensions/sql-migration/package.json

```diff
@@ -143,6 +143,18 @@
         }
       ]
     },
+    "configuration": [
+      {
+        "title": "%migration-dashboard-title%",
+        "properties": {
+          "azureRecommendation.dataCollectionPath": {
+            "type": "string",
+            "default": "%userprofile%\\AppData\\Roaming\\azuredatastudio\\PerfData",
+            "description": "Default path to store collected performance data for SKU recommendations."
```

Comment:
But these should be localized, if at all this is something we need to discuss with ADS team, let's start a thread with them.

Reply:
We can confirm this with the ADS team whether this needs to be localized or not. For now, I think we can go with this, since it is same across extensions

Reply:
please localize all user readable strings.

Reply:
Done. Localized the description as well.